src/lib/draw/route/route_tool.ts

```diff
@@ -94,6 +98,8 @@ export class RouteTool {
       this.inner.onMouseMove(e.lngLat.lng, e.lngLat.lat, circleRadiusMeters)
     ) {
       this.redraw();
+      // TODO We'll call this too frequently
```

Comment:
This one is a bit spammy. Every time you mouseover a waypoint, it'll act like an edit, even if nothing has changed yet. I'll fix this upstream in route-snapper and update here later.